forte/solvers/mcscf.py

```diff
@@ -0,0 +1,132 @@
+from forte.core import flog
+
+from forte.solvers.feature import Feature
+from forte.solvers.solver import Solver
+
+from forte.forte import ForteOptions
+from forte import forte_options
+from forte.forte import make_mcscf_two_step, make_mo_space_info_from_map
+
+
+class MCSCF(Solver):
+    """
+    A class to perform orbital optimization
```

Comment:
This docstring has me thinking: can I use this class to do OMP2, or is it just a class to do orbital optimized determinant-based computations?